lib/cable_ready/updatable/memory_cache_debounce_adapter.rb

```diff
@@ -0,0 +1,22 @@
+module CableReady
+  module Updatable
+    class MemoryCacheDebounceAdapter
+      include Singleton
+
+      delegate_missing_to :@store
+
+      def initialize
+        super
+        @store = ActiveSupport::Cache::MemoryStore.new(expires_in: 5.minutes)
```

Comment:
> This cache has a bounded size specified by the :size options to the initializer (default is 32Mb).

I'd recommend that we limit max size even further to something like 8Mb or 16Mb. At the very least, I'd like to make max size explicit so it's more obvious. Thoughts?

Reply:
Yeah totally. I also thought to make it configurable, but at present this is probably an overdo.

I think 8Mb will do.